docs/getting-started/install-cent-node.md

```diff
@@ -69,52 +97,35 @@ Once you have installed all the necessary packages, follow these steps to instal
   -a 8082 -p 38204
   ```
 
-  This command automatically adds the Identity keys to the `config.yaml` file.
+  This command automatically creates an Identity and keys needed and generates the `config.yaml` file required to run the node.
```

Comment:
This command automatically creates an Identity and the key pairs. It then generates the `config.yaml` file required to run the node.